vet.sh

```diff
@@ -94,7 +94,7 @@ git grep '"github.com/envoyproxy/go-control-plane/envoy' -- '*.go' | not grep -v
 go list -f {{.Dir}} ./... | xargs go run test/go_vet/vet.go
 
 # - gofmt, goimports, golint (with exceptions for generated code), go vet.
-gofmt -s -d -l . 2>&1 | fail_on_output
+gofmt -s -d . 2>&1 | fail_on_output
```

Comment:
Just curious, what does this `-l` do? Why do we want to remove it?

Reply:
Oops, I removed it so when I run the vet script locally there would be helpful debugging output and forgot to add it back. Going to do that now.